handlers/authorization/auth.py

```diff
@@ -4,31 +4,30 @@
 from schemas.user import AuthOut
 
 from sqlalchemy import select
-from fastapi import status, HTTPException, Depends, APIRouter
+from fastapi import Depends, APIRouter
 from fastapi.security import OAuth2PasswordRequestForm
 
+from components.exceptions import BackendException
+
 router = APIRouter()
 
 
-@router.post('/login', summary="Створення доступу та оновлення токена користувача", response_model=AuthOut,
-             tags=["Authorization"])
+@router.post('/login', summary="Створення доступу та оновлення токена користувача", response_model=AuthOut, tags=["Authorization"])
 async def login(form_data: OAuth2PasswordRequestForm = Depends()):
     query = select(User).where(User.login == form_data.username)
     user = await database.fetch_one(query)
     if not user:
-        raise HTTPException(
-            status_code=status.HTTP_400_BAD_REQUEST,
-            detail="Некоректний логін або пароль. Будь ласка, спробуйте ще раз."
+        raise BackendException(
+            message="Login or password is invalid. Please, try again."
         )
 
     hashed_pass = user.password
 
     if not verify_password(form_data.password, hashed_pass):
-        raise HTTPException(
-            status_code=status.HTTP_400_BAD_REQUEST,
-            detail="Невірна електронна адреса або пароль."
+        raise BackendException(
+            message="Email or password is invalid. Please, try again."
         )
-
+    
```

Comment:
Line should be blank. [From PEP-8](https://pep8.org/#blank-lines):

> Use blank lines in functions, sparingly, to indicate logical sections.

Reply:
I couldn't understand what to do. Add another one or delete that present in code?

Reply:
Neither add nor delete any lines. Just don't change (do not add redundant spaces)